CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -1593,10 +1593,16 @@ public override void OnApplyTemplate()
         {
             base.OnApplyTemplate();
 
-            // Create main window
-            Content = image = CreateImage();
+            if (image == null)
```

Comment:
It is, look at https://github.com/cefsharp/CefSharp/pull/1947#issuecomment-280770217